projects/Mallard/src/screens/settings/default-info-text-webview.tsx

```diff
@@ -39,6 +40,7 @@ const DefaultInfoTextWebview = ({ html }: { html: string }) => {
                 originWhitelist={['*']}
                 source={{ html: makeHtml({ styles, body: html }) }}
                 style={{ flex: 1 }}
+                useWebKit={false}
```

Comment:
I was looking into this issue the other way and was confused why fonts wouldn't work, but Rich did mention that little line was the secret 😅

Reply:
hahaha, it's not ideal but it works for now! its an issue in native iOS too